reconcile/change_owners/change_log_tracking.py

```diff
@@ -25,6 +28,7 @@
 @dataclass
 class ChangeLogItem:
     commit: str
+    created_at: str
```

Comment:
should we use proper timestamps here? string can lead to unforeseen parsing issues long-term.

Reply:
E.g., in the past we had issues with string https://github.com/app-sre/qontract-reconcile/pull/4525

Reply:
this is consistent with every other gitlab usage across the code base

Reply:
example api response: https://gitlab.cee.redhat.com/api/v4/projects/13582/repository/commits/27fbcf9f76e47a98a17ba146fe0f86e6e608a108